exercises/triangle/triangle_test.py

```diff
@@ -3,55 +3,45 @@
 from triangle import Triangle, TriangleError
 
 
+# Tests adapted from `problem-specifications/exercises/\
+# triangle/canonical-data.json` @ v1.0.0
```

Comment:
This version string should be: ```# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0```

We leave the intervening paths as being implied by the `//` to save splitting the line as you had to do.